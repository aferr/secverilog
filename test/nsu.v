module nsu(
	   input {H} h1,
	   input {L} clk
	   );

   reg seq {LH x} x;
   reg seq {L} l1;
   reg seq {L} l2;   
   
   always@(posedge clk) begin
      if (h1) begin
	 x <= 1;
      end
      if (x == 0 && l1 == 1) begin
	 l1 <= 0;	 
      end
      l2 <= 1;      
   end
   
endmodule
