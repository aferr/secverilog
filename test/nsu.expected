unsat
Start base conditions are satisfiable? (should be sat)
sat
End base conditions are satisfiable?
unsat
sat
unsat
unsat
